tests/unit/customizations/emr/test_set_unhealthy_node_replacement.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from tests.unit.customizations.emr import EMRBaseAWSCommandParamsTest as \
+    BaseAWSCommandParamsTest
```

Comment:
I'd recommend either 1) Not renaming `EMRBaseAWSCommandParamsTest` or 2) Just use `awscli.testutils.BaseAWSCommandParamsTest` directly. Mainly it is confusing to override the name of the child class to it's parent class at import. I don't have a strong preference on which option